src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +42,34 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${selectedTodo.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
Try to rewrite the code so that it is not necessary to disable the linter

Reply:
@roman-mirzoian  I can't fix it either